package mypack.example;

public class Aithmetic_Exception {

    public static void main(String[] args){
//    try{
//        int a=100;
//        int b=0;
//        int c=a/b;         //Arithmetic Exception occurs
//        System.out.println(a+b);
//    }catch(Exception a){
//        System.out.println("Exception occurs");
//    }
//    finally {         //this block always print if exception occur or not
//        System.out.println("Exception Handle");
//    }
    }
}
